examples/getting-started-movielens/04a-Triton-Inference-with-TF.ipynb

```diff
@@ -552,72 +550,72 @@
-      " [0.6154156 ]\n",
-      " [0.62231386]\n",
-      " [0.63039726]\n",
-      " [0.64072984]\n",
-      " [0.61164194]\n",
-      " [0.64943635]\n",
-      " [0.609999  ]\n",
-      " [0.6285001 ]\n",
-      " [0.6202306 ]\n",
-      " [0.6135287 ]\n",
-      " [0.6249209 ]\n",
-      " [0.61371297]\n",
-      " [0.6129487 ]\n",
-      " [0.62337595]\n",
-      " [0.63045645]\n",
-      " [0.62860096]\n",
-      " [0.62110895]\n",
-      " [0.61858165]\n",
-      " [0.61338454]\n",
-      " [0.6230694 ]\n",
-      " [0.62595254]\n",
-      " [0.6108157 ]\n",
-      " [0.61079603]\n",
-      " [0.6127178 ]\n",
-      " [0.62006015]\n",
-      " [0.62082994]\n",
-      " [0.6235482 ]\n",
-      " [0.62835073]\n",
-      " [0.62865674]\n",
-      " [0.6215745 ]\n",
-      " [0.6171293 ]\n",
-      " [0.62446827]\n",
-      " [0.6171792 ]\n",
-      " [0.61890376]\n",
-      " [0.6285598 ]\n",
-      " [0.61814624]\n",
-      " [0.6170676 ]\n",
-      " [0.6135012 ]\n",
-      " [0.6257643 ]\n",
-      " [0.6276466 ]\n",
-      " [0.6206201 ]\n",
-      " [0.62115675]\n",
-      " [0.61478084]] \n",
+      " [[0.63028914]\n",
+      " [0.6134343 ]\n",
+      " [0.60138303]\n",
+      " [0.6282814 ]\n",
+      " [0.6238409 ]\n",
+      " [0.6414971 ]\n",
+      " [0.6275666 ]\n",
+      " [0.6143182 ]\n",
+      " [0.62991506]\n",
+      " [0.6175872 ]\n",
+      " [0.62239087]\n",
+      " [0.62810206]\n",
+      " [0.641524  ]\n",
+      " [0.62629336]\n",
+      " [0.6398284 ]\n",
+      " [0.6248835 ]\n",
+      " [0.61815184]\n",
+      " [0.62088364]\n",
+      " [0.6301337 ]\n",
+      " [0.6178366 ]\n",
+      " [0.6184469 ]\n",
+      " [0.6224728 ]\n",
+      " [0.62434137]\n",
+      " [0.6256087 ]\n",
+      " [0.63201374]\n",
+      " [0.61148345]\n",
+      " [0.6406433 ]\n",
+      " [0.61648977]\n",
+      " [0.62570417]\n",
+      " [0.62155634]\n",
+      " [0.60143846]\n",
+      " [0.6250419 ]\n",
+      " [0.60258746]\n",
+      " [0.61976403]\n",
+      " [0.6100665 ]\n",
+      " [0.61903626]\n",
+      " [0.6186965 ]\n",
+      " [0.61972004]\n",
+      " [0.61375433]\n",
+      " [0.62594193]\n",
+      " [0.62070924]\n",
+      " [0.6245276 ]\n",
+      " [0.6208814 ]\n",
+      " [0.6172946 ]\n",
+      " [0.60161334]\n",
+      " [0.6165426 ]\n",
+      " [0.6099649 ]\n",
+      " [0.6209466 ]\n",
+      " [0.6253646 ]\n",
+      " [0.6335443 ]\n",
+      " [0.61978215]\n",
+      " [0.6162915 ]\n",
+      " [0.63050246]\n",
+      " [0.6245917 ]\n",
+      " [0.6182605 ]\n",
+      " [0.63328594]\n",
+      " [0.6265736 ]\n",
+      " [0.61479115]\n",
+      " [0.5996266 ]\n",
+      " [0.6261515 ]\n",
+      " [0.6272384 ]\n",
+      " [0.6188991 ]\n",
+      " [0.623684  ]\n",
+      " [0.62370366]] \n",
       "\n",
-      "run_time(sec): 0.06536054611206055 - rows: 64 - inference_thru: 979.1839849421103\n"
+      "run_time(sec): 0.8563857078552246 - rows: 64 - inference_thru: 74.73268109563017\n"
```

Comment:
The runtime result is very high, and throughput is very low. In the inference nb, we have runtime as `0.065` sec and throughput as `979`.  

Runtime `0.85` sec is pretty high with very low throughput, 74. It is  13 times lower. This might be due to running the inference nb on an NGC instance?

Reply:
Removed the notebook so Ronay can merge the updated one